### ISSUE TITLE
Fixed arc for Macro Bug

### ISSUE THREAD

**Issue by Bedrettin-Cetinkaya** [melodyguan/enas#56]
I think, there is bug in general_child.py file. Even if fixed_arc contains separable convolution operations, _fixed_layer function adds normal convolution , not separable.

**Comment by unlabeledData** [comment]
@Bedrettin-Cetinkaya Do you fixed this bugs?  It looks like you also use standard convolution instead of separable convolution in your keras ENAS project. Could you tell me the reason?

**Comment by Bedrettin-Cetinkaya** [comment]
@unlabeledData ,I think, purpose of using separable convolution is decreasing number of parameters ,traning time ,complexity for model exploration step and there is no  other advantage. Therefore, after found the best model, it is changed to normal convolution operation.

**Comment by Bedrettin-Cetinkaya** [comment]
@unlabeledData ,If i undestand your question correctly, you ask that why doesn't  "cifar10_macro_final.sh" include any layer with index 4,5 , respectively average pooling and max pooling layer? 

For my output architecture, i chose best model for my problem, and it was just an example. I didn't consider that whether it includes any pooling or doesn't, i just considered model accuracy. Also, there may be many reason for not using pooling layers. It depends on your hyperparameter settings, dataset, "macro_search.sh", settings, etc.
